docs/source/upcoming_release_notes/385-Hutch-python_feature:_auto-close_after_timeout.rst

```diff
@@ -0,0 +1,22 @@
+385 Hutch-python feature: auto-close after timeout
+#################
+
+API Changes
+-----------
+- N/A
+
+Features
+--------
+- N/A
```

Comment:
We need to fill out these sections to help us set up the release notes later down the line